### ISSUE TITLE
The weight decay (the regularization loss) is not applied for the fc_layer and upscore_layer

### ISSUE THREAD

**Issue by SpinachR** [MarvinTeichmann/tensorflow-fcn#58]
Hello!
I found that in fcn8s_vgg.py. The weight decay loss is not added into tf.GraphKeys.REGULARIZATION_LOSSES, since you specify another collection name.

I would like to know if this will affect the final results?
